adafruit_us100.py

```diff
@@ -37,6 +37,10 @@ class US100:
 
     def __init__(self, uart: UART) -> None:
         self._uart = uart
+        # Some chips, such as ESP32-S3, may have junk input bytes immediately after UART creation.
+        # Wait enough time for those to appear, and then clear the buffer.
+        time.sleep(0.1)
+        uart.reset_input_buffer()
```

Comment:
I have 2 concerns about this:
  - if this is ESP32-S3 specific, this will unnecessarily delay other HW
  - if the issue is specific to UART (on ESP32-S3) rather than US-100, the fix should probably reside in CP itself

Reply:
The delay only happens once, in the constructor. I could probably make this 2 msecs instead of 100 msecs. I can test that.

Yes, ESP32-S3 should be fixed. I am looking at that. But this makes the library more robust for any case like this where it might get out of sync. And it makes the library usable without having to upgrade CircuitPython.

Reply:
That's a fair argument, esp. the second part.